sections/06_discussion.md

```diff
@@ -198,6 +198,124 @@ influence healthcare decisions? Or, is deep learning a hypothesis generation
 machine that requires manual validation? DeepChem and DragoNN are worth
 discussing here.*
 
-### Transfer learning/transferability of features
+### Multimodal and transfer learning
 
 * https://github.com/greenelab/deep-review/issues/139#issuecomment-268901804
+
+As discussed above, the fact that biomedical datasets often contain a limited
+number of instances or labels, often leads to poor performance of
+machine learning algorithms. When trained on such datasets, deep learning
+models are particularly prone to overfitting due to their high representational
+power. However, Transfer Learning techniques also known as Domain Adaptation
+enable transfer of extracted patterns between different datasets and even
+domains. This allows a model to take advantage on larger labeled data for
+training of a model which can  then can be reused for the problem in hand. Deep
+neural networks exhibit high transferability of learnt features including when
+pre-training and target sets are distant [@tag:Yosinski2014].
+
+In image analysis, previous examples of deep transfer learning applications
+proved large scale natural image sets [@tag:Russakovsky2015_imagenet]
+to be useful for pre-training models that can then serve as generic feature
+extractors applied to various types of biological images
+[@tag:Zeng2015 @tag:Angermueller2016_dl_review @tag:Pawlowski2016]. More
+recently, deep learning models trained to predict protein sub-cellular
+localization successfully performed predictions for  proteins
+ not originally present in the training set [@tag:Parnamaa2017].
+Moreover, in this type of task, learnt features performed reasonably well even
+when applied to images obtained using different fluorescent labels, imaging
+techniques, and different cell types [@tag:Kraus2017_deeploc]. However, there
+are no established theoretical guarantees for feature transferability between
+such distant domains such as natural images and various modalities of biological
+imaging. Because learnt patterns are represented in deep neural networks in a
+layer-wise hierarchical fashion, this issue is usually addressed by fixing an
+empirically chosen number of layers that preserve generic characteristics
+of both training and target datasets. Then, the model is fine-tuned by
+re-training networks' top layers on the specific dataset in order to
+re-learn domain-specific high level concepts. Fine-tuning on specific biological
+datasets enables more focused predictions. For example, Min et al.
```

Comment:
It might be worth also mentioning Basset ( https://github.com/greenelab/deep-review/issues/16 ) by @davek44 to make clear that this type of transfer learning seems to be generally applicable. In Basset the authors demonstrate speedy fine-tuning of a general model.